argostime/static/graphs.js

```diff
@@ -92,5 +97,9 @@ for (var i = 0; i < graphDivs.length; i++) {
         });
         xhr.open("GET", `/productoffer/${offer}/price_step_graph_data.json`);
         xhr.send();
+
+        window.addEventListener('resize', function(event) {
```

Comment:
Waar is dit precies voor?

Reply:
Ik heb de viewport width nodig in CSS voor de scale transform. Nu is dit gewoon beschikbaar, `100vw` is de viewport width in pixels. Echter mag je geen pixels in een scale transform gebruiken en daarom moeten we dit doen om het als number type te krijgen.